### DISCUSSION TITLE
synchronise eldoc and corfu next/previous?

### DISCUSSION THREAD

**Discussion by yatesco** [minad/corfu#385] [Q&A]
Hi there, is there anyway I can have the `*eldoc*` buffer show the documentation for the currently selected `corfu` item? I realise I can use https://github.com/minad/corfu/blob/main/extensions/corfu-popupinfo.el but I quite like using `eldoc` for this.

Thanks!

**Comment by minad** [comment]
There is also corfu-info.el. Did you try this? There is no additional Eldoc integration in Corfu.

**Reply by yatesco** [reply]
thanks! - I'll look into that.
